alerter.js

```diff
@@ -1,26 +1,39 @@
+const { expect } = require("chai");
 let alertFailureCount = 0;
+let threshold = 200;
 
 function networkAlertStub(celcius) {

```

Comment:
The stub is in the same file as the functionality. This means it will get delivered along with the product...
Separate the stub into its own file.